app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardCSVStep/CSVDropzoneField.js

```diff
@@ -77,52 +95,77 @@ class CSVDropzoneField extends React.Component {
       <ListView>
         {value.map(vm => (
           <ListView.Item
+            key={vm.reference}
             description={vm.name || ''}
             additionalInfo={[
-              <ListView.InfoItem>
+              <ListView.InfoItem key="provider">
                 {`${__('Provider:')} ${vm.provider || ''}`}
               </ListView.InfoItem>,
-              <ListView.InfoItem>
+              <ListView.InfoItem key="reference">
                 {`${__('Reference:')} ${vm.reference || ''}`}
               </ListView.InfoItem>
             ]}
           />
         ))}
       </ListView>
     );
-    const results = csvDropped && (
+    const results = csvImported && (
       <Toolbar.Results>
         {!error && value && sprintf(__('%s Items'), value.length)}
         {pristine && error}
       </Toolbar.Results>
     );
     return (
-      <Dropzone
-        accept=".csv"
-        className={cx('csv-upload-dropzone')}
-        activeClassName={cx('active')}
-        onDrop={this.onFileDrop}
-        disableClick
-        disablePreview
-        ref={node => {
-          dropzoneRef = node; // See comment at top, this is so we can call dropzoneRef.open below
-        }}
-      >
-        <Toolbar>
-          <Toolbar.RightContent>
-            <Button
-              bsStyle="primary"
-              onClick={() => {
-                dropzoneRef.open();
-              }}
-            >
-              {__('Browse...')}
-            </Button>
-          </Toolbar.RightContent>
-          <div className="form-group">{mainContents}</div>
-          {results}
-        </Toolbar>
-      </Dropzone>
+      <React.Fragment>
+        <Dropzone
+          accept=".csv"
+          className="csv-upload-dropzone"
+          activeClassName="active"
+          onDrop={this.onFileDrop}
+          disableClick
+          disablePreview
+          ref={node => {
+            dropzoneRef = node; // See comment at top, this is so we can call dropzoneRef.open below
+          }}
+        >
+          <Toolbar>
+            <Toolbar.RightContent>
+              <Button bsStyle="primary" onClick={dropzoneRef.open}>
```

Comment:
Not sure why this is a problem, but if I change this back to 
```onClick={() => { dropzoneRef.open() }}```
the error I mention goes away

Reply:
Oh, duh. it's because `dropzoneRef.open` is not bound properly. @priley86, i needed that anonymous function, I knew it! Sorry I didn't test after changing this, guys, i thought it was trivial / functionally no change.

Reply:
Oh right if it’s not bound I agree with you ;) ... fine w/ anonymous in that case then ...